boot/freeldr/freeldr/arch/i386/i386bug.c

```diff
@@ -115,9 +116,11 @@ i386PrintExceptionText(ULONG TrapIndex, PKTRAP_FRAME TrapFrame, PKSPECIAL_REGIST
     i386_ScreenPosX = 0;
     i386_ScreenPosY = 0;
 
-    PrintText("An error occured in " VERSION "\n"
+    PrintText("Error in %s %s\n"
               "Report this error on the ReactOS Bug Tracker: https://jira.reactos.org\n\n"
-              "0x%02lx: %s\n\n", TrapIndex, i386ExceptionDescriptionText[TrapIndex]);
+              "0x%02lx: %s\n\n",
+              KERNEL_VERSION_STR, KERNEL_VERSION_BUILD_STR,
```

Comment:
Your new current code would show: "Error in 0.4.15 20240823-blablah" ...

Reply:
This is exactly what we do show in cmd, and in ntoskrnl. We do show both:
(ntoskrnl/kd64/kdinit.c:75) ReactOS 0.4.15-x86-dev (Build 20240816-0.4.15-dev-8532-g6e86daa)